srtcore/buffer.h

```diff
@@ -60,6 +60,19 @@ modified by
 #include "utilities.h"
 #include <fstream>
 
+// The notation used for "circular numbers" in comments:
+// The "cicrular numbers" are numbers that when increased up to the
+// maximum become zero, and similarly, when the zero value is decreased,
+// it turns into the maximum value minus one. This wrapping works the
+// same for adding and subtracting. Circular numbers cannot be multiplied.
+
+// Operations done on these numbers are marked with additional % character:
+// a %> b : a is later than b
+// a ++% (++%a) : shift a by 1 forward
+// a +% b : shift a by b
+// a == b : equality is same as for just numbers
+
+
```

Comment:
Thankfully, I don't see this notation used anywhere in the comments. Also, I think the is absolutely no need to explain modulo operations or invent new names or notations for them. Could we please kill this entire comment section?

Reply:
This notation is to help people reading the code understand what is being done inside. Developers do need it. If you want, I can remove it from here, but it will be soon brought back through another PR.

Reply:
Yes, please. As it stands, the comment is not helpful, especially to developers. Will see how it looks in the context of the future PR. Thanks.

Reply:
See fec.cpp line 1731. Still, there are more such comments in the incoming branches.